src/seq_analysis/orf.rs

```diff
@@ -93,16 +93,18 @@ pub struct Orf {
 
 /// The current algorithm state.
 struct State {
-    start_pos: [Option<usize>; 3],
+    start_pos: [Vec<usize>; 3],
```

Comment:
Since there can now be 0 or more active start positions (instead of just 0 or 1), we move to `Vec` instead of `Option`